man/man5/zfs-module-parameters.5

```diff
@@ -2552,12 +2552,13 @@ Default value: \fB0\fR.
 .sp
 .ne 2
 .na
-\fBzio_delay_max\fR (int)
+\fBzio_slow_io_ms\fR (int)
 .ad
 .RS 12n
-A zevent will be logged if a ZIO operation takes more than N milliseconds to
-complete. Note that this is only a logging facility, not a timeout on
-operations.
+When an I/O operation takes more than \fBzio_slow_io_ms\fR milliseconds to
+complete is marked as a slow I/O.  Each slow I/O causes a delay zevent.  Slow
```

Comment:
This talks about `delay zevent`, and `zfs_slow_ios_per_second` talks about `slow I/O (delay) events`.  I think these are talking about the same thing, so we should use the same term in both places.

Reply:
They're close to the same thing.   "slow" refers to IOs, "delay" to the event. So an IO is marked as "slow" which in turn causes a "delay event". Technically `zfs_slow_ios_per_second` should be called `zfs_delays_per_second`, since it's rate-limiting the events, but we wanted to avoid confusing it with TXG delays, so I kept it as `zfs_slow_ios_per_second`.

Reply:
I think both of these are referring to events (not the i/o itself).  I would suggest you change the `zfs_slow_ios_per_second` description to say:
```
Rate limit delay zevents (which report slow i/os) to this many per second.
```
That way both places are using the term `delay zevents` to refer to the event.